sdk/core/src/CommonNetworkMember.ts

```diff
@@ -632,6 +635,16 @@ export class CommonNetworkMember {
 
     const { userPoolId, clientId } = CommonNetworkMember.setEnvironmentVarialbles(options)
 
+    if (messageParameters) {
+      const customMessagesTemplateService = new CustomMessageTemplatesService(options)
```

Comment:
Is it "custom messages" or "custom message"? :)

Reply:
And we already have `this._customMessageTemplateService` at this point, don't we?
(Which also does not seem to be used anywhere)

Reply:
Agreed @Volland - lets have one convention:
customMessage`s`TemplateService = new CustomMessageTemplatesService(options)